services/keywords/utils.py

```diff
@@ -0,0 +1,60 @@
+from google.cloud import language # pylint: disable=import-error
+from google.cloud.language import enums, types # pylint: disable=import-error
+from google.api_core import exceptions
+import constants
+
+def gcloud_syntax_extraction(text):
+    """
+    Calls the gcloud natural langauge api's analyze syntax
+    Attributes:
+        text: the text which we wish to analyze
+    Returns:
+        the response given by the api call
+    """
+    try:
+        client = language.LanguageServiceClient()
+        # pylint: disable=no-member
+        document = types.Document(
+            content=text,
+            type=enums.Document.Type.PLAIN_TEXT)
+        response = client.analyze_syntax(document=document)
+    except exceptions.GoogleAPICallError as err:
+        return {"error": {"type": "GoogleAPICallError", 
+            "message": getattr(err, 'message', str(err))}}
+    except TypeError as err:
+        return {"error": {"type": "TypeError", 
+            "message": getattr(err, 'message', str(err))}}
+    except ValueError as err:
+        return {"error": {"type": "ValueError", 
+            "message": getattr(err, 'message', str(err))}}
+
+
+    return response
+
+def extract_keywords(text):
```

Comment:
[nit] This is pretty much your core service logic vs. just a util func. Maybe move this to `main.py` or rename this file to something more specific, like `extract.py`